api/sse/tasks.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+import requests
+from django.conf import settings
+
+from task_processor.decorators import register_task_handler
+
+
+@register_task_handler()
+def send_environment_update_messages(environment_keys: List[str]):
```

Comment:
This is probably `typing.Iterable`, right? I think sometimes we're passing a queryset.

Reply:
No, we are not any more